README.md

```diff
@@ -135,35 +135,81 @@ const exampleBlockedTiersInDaysData = {
 }
 ```
 
-**DemandTiersInMonths**: A blocked rate structure, based on the max demand for every month. Accepts load profile filters to model seasonal rates.
+**Demand**: A flexible demand rate structure based on peak load, with tiering, averaging, and load profile filters.
 
+The `Demand` billing determinant computes values based on maximum demand within a specified period (e.g. monthly, daily, annual). It supports:
+
+* Tiered billing determinants, with per-month min/max values for each tier
+
+* Averaging, such as taking the top 3 daily peaks per month
+
+* Filtering, to include only specific months, days, hours, or dates
+
+Basic Example – Monthly Demand with Seasonal Filter
+```js
+const demand = {
+  rateElementType: RateElementTypeEnum.Demand,
+  name: 'Winter-Only Demand Charge',
+  demandPeriod: 'monthly',
+  filters: {
+    months: [0, 1, 2, 3, 4, 9, 10, 11],
+  },
```

Comment:
i actually didn't intend for this interface to require an explicit `filters` or `options` keys...

matching the way the other rate elements work would be:

```javascript
const demand = {
  rateElementType: RateElementTypeEnum.Demand,
  name: 'Winter-Only Demand Charge',
  demandPeriod: 'monthly',
  months: [0, 1, 2, 3, 4, 9, 10, 11], // no need to nest under filters:
  averagingPeriod: 'monthly',
  averagingQty: 3,
},
```

Is that possible to achieve here?

Reply:
I'm coming at this from the "match the existing pattern in other billing determinant strategies" perspective, but since this rate element is breaking patterns a bit I'm also willing to consider that these things _should_ be nested.

Reply:
Makes sense! I was a bit confused by the `options`, it's easy to make it match other billing determinants